x/ecocredit/migrations/v3/patch.go

```diff
@@ -0,0 +1,191 @@
+	//  C01-20190101-20191231-009  ->  "2022-05-11T11:35:30Z"
+	//  C01-20180909-20200101-004  ->  "2022-03-08T17:25:23Z"
+	//  C01-20180101-20200101-001  ->  "2022-02-09T09:10:02Z"
+	//  C01-20170101-20180101-007  ->  "2022-03-30T15:04:30Z"
+	//  C01-20170101-20180101-006  ->  "2022-03-30T07:51:12Z"
+	//  C01-20170101-20180101-005  ->  "2022-03-30T07:46:01Z"
+	//  C02-20200909-20210909-001  ->  "2022-03-08T13:00:50Z"
+	//  C02-20210909-20220101-002  ->  "2022-03-08T17:17:20Z"
+	//  C04-20180202-20190202-001  ->  "2022-03-28T08:31:45Z"
+	//  C04-20190202-20200202-002  ->  "2022-03-28T08:45:14Z"
+
+	batchIdToIssuanceDateMap := make(map[string]string)
+	batchIdToIssuanceDateMap["C01-20210909-20220101-003"] = "2022-03-08T17:18:19Z"
+	batchIdToIssuanceDateMap["C01-20190101-20210101-008"] = "2022-04-22T16:32:09Z"
+	batchIdToIssuanceDateMap["C01-20190101-20210101-002"] = "2022-02-14T09:07:25Z"
+	batchIdToIssuanceDateMap["C01-20190101-20191231-009"] = "2022-05-11T11:35:30Z"
+	batchIdToIssuanceDateMap["C01-20180909-20200101-004"] = "2022-03-08T17:25:23Z"
+	batchIdToIssuanceDateMap["C01-20180101-20200101-001"] = "2022-02-09T09:10:02Z"
+	batchIdToIssuanceDateMap["C01-20170101-20180101-007"] = "2022-03-30T15:04:30Z"
+	batchIdToIssuanceDateMap["C01-20170101-20180101-006"] = "2022-03-30T07:51:12Z"
+	batchIdToIssuanceDateMap["C01-20170101-20180101-005"] = "2022-03-30T07:46:01Z"
+	batchIdToIssuanceDateMap["C02-20200909-20210909-001"] = "2022-03-08T13:00:50Z"
+	batchIdToIssuanceDateMap["C02-20210909-20220101-002"] = "2022-03-08T17:17:20Z"
+	batchIdToIssuanceDateMap["C04-20180202-20190202-001"] = "2022-03-28T08:31:45Z"
+	batchIdToIssuanceDateMap["C04-20190202-20200202-002"] = "2022-03-28T08:45:14Z"
+
+	// update issuance date for credit batches
+	if err := updateBatchIssueanceDate(ctx, ss, oldBatchDenomToNewDenomMap, batchIdToIssuanceDateMap); err != nil {
+		return err
+	}
+
+	// basket name to curator
+	// rNCT   -> regen1df675r9vnf7pdedn4sf26svdsem3ugavgxmy46
+	// NCT    -> regen1df675r9vnf7pdedn4sf26svdsem3ugavgxmy46
+
+	basketNameToCuratorMap := make(map[string]string)
+	basketNameToCuratorMap["rNCT"] = "regen1df675r9vnf7pdedn4sf26svdsem3ugavgxmy46"
+	basketNameToCuratorMap["NCT"] = "regen1df675r9vnf7pdedn4sf26svdsem3ugavgxmy46"
+	if err := updateBasketCurator(ctx, ss, basketStore, basketNameToCuratorMap); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func updateBasketCurator(ctx context.Context, ss api.StateStore, basketStore basketapi.StateStore,
+	basketNameToCuratorMap map[string]string) error {
+	for name, curator := range basketNameToCuratorMap {
+		basket, err := basketStore.BasketTable().GetByName(ctx, name)
+		if err != nil {
+			return err
+		}
+		basket.Curator = sdk.AccAddress(curator)
+		if err := basketStore.BasketTable().Update(ctx, basket); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func patchMainnet(ctx context.Context, ss api.StateStore, oldBatchDenomToNewDenomMap map[string]string) error {
```

Comment:
Might be nice to move this between `pathMigrations` and `patchRedwood` so that the order is:

- `patchMigrations`
- `patchMainnet`
- `patchRedwwod`

And then then helper functions below.